### ISSUE TITLE
Feng office 3.10.4.3 throws 'Found faulty query' error

### ISSUE THREAD

**Issue by prbt2016** [fengoffice/fengoffice#75]
Hello @gdasanbiagio ,

I was in the process of manual installation of Feng office on Centos7 with  PHP 7.1, MYSQL 5.7, Apache 2.2.
However on final step i.e step 4 of installation I get this error . 

```
Found faulty query: -- option Minimum number of characters for dimension search INSERT INTO `fo_config_options` (`category_name`, `name`, `value`, `config_handler_class`, `is_system`, `option_order`, `dev_comment`, `options`) VALUES ('general', 'minimum_characters_dimension_search', '3', 'IntegerConfigHandler', '0', '0', 'Minimum number of characters for dimension search', '');
Failed to import initial data. MySQL said: Table '{{DBNAME}}.fo_config_options' doesn't exist
```

As shown in the screenshot below : 

![image](https://github.com/fengoffice/fengoffice/assets/17871448/bd980c3d-cda8-4a88-8b0d-104bff9e883a)

I haven't used the table prefix fo_ . Seems that table prefix fo_ is hardcoded , instead of taking dynamically in file : 

public/install/installation/templates/sql/mysql_initial_data.php on line 1393 i.e here  : 

[Line 1393](https://github.com/fengoffice/fengoffice/blob/master/public/install/installation/templates/sql/mysql_initial_data.php#L1393)

ie :
INSERT INTO `fo_config_options` (`category_name`, `name`, `value`, `config_handler_class`, `is_system`, `option_order`, `dev_comment`, `options`)

Can you please resolve this by merging the pull request I have made here  #76

**Comment by prbt2016** [comment]
Hello ,

@gdasanbiagio 

Any update for the same ?.

**Comment by SantiFeng** [comment]
This issue has been already fixed and merge in version 3.10.7.9

**Comment by SantiFeng** [comment]
This issue has been already fixed and merge in version 3.10.7.9
